apps/state/lib/state/trip/added.ex

```diff
@@ -52,10 +52,10 @@ defmodule State.Trip.Added do
   end
 
   @spec prediction_to_trip({Trip.id(), Prediction.t()}) :: [Trip.t()]
-  defp prediction_to_trip({trip_id, %{route_pattern_id: route_pattern_id} = prediction})
-       when is_binary(route_pattern_id) do
-    with %{representative_trip_id: rep_id} <- State.RoutePattern.by_id(route_pattern_id),
-         [trip | _] <- State.Trip.by_id(rep_id) do
+  defp prediction_to_trip({trip_id, prediction}) do
+    with %{route_pattern_id: route_pattern_id} when not is_nil(route_pattern_id) <- prediction,
```

Comment:
Why the change from `is_binary(route_pattern_id)` to `not is_nil(route_pattern_id)`?

Reply:
Ah, that was part of an intermediate refactoring step and I didn't notice it was kind of a pointless change now. It should be functionally equivalent, since the value can only be either a string or `nil`, but I'll change it back.